Scripts/cmake/o3de_package_utilities.cmake

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+include(CMakeParseArguments)
```

Comment:
CMakeParseArguments is not needed since [CMake 3.04](https://cmake.org/cmake/help/latest/module/CMakeParseArguments.html?highlight=cmakeparsearguments)

Reply:
Weill remove.  Although its harmless, so I'm probably not going to rev the package itself for this.